README.md

```diff
@@ -45,6 +45,63 @@ implemented and how they can be derived from new methods.
     - Two promises are equivalent when they yield equivalent values.
     - Two functions are equivalent if they yield equivalent outputs for equivalent inputs.
 
+## How to add Fantasy Land compatibility to your library
+
+1. Add `fanatasy-land` package as a peer dependency. Add in your `package.json`:
```

Comment:
Since we're using string values, could one use a regular dependency rather than a peer dependency?